sys/net/gnrc/transport_layer/tcp/gnrc_tcp_option.c

```diff
@@ -64,6 +64,9 @@ int _option_parse(gnrc_tcp_tcb_t *tcb, tcp_hdr_t *hdr)
                 DEBUG("gnrc_tcp_option.c : _option_parse() : Unknown option found.\
                       KIND=%"PRIu8", LENGTH=%"PRIu8"\n", option->kind, option->length);
         }
+        if (option->length > opt_left) {
+            return 0;
```

Comment:
Maybe add a DEBUG statement?

Reply:
Since the `default` case generates a `Unknown option found` one, this should too.

Reply:
Ok, done. Thanks for the feedback!